src/components/uyghur/boycott/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+
+// semantic-ui
+import {
+    Container, Icon, Label, Menu, Table 
+  } from 'semantic-ui-react';
+
+class UyghurBoycott  extends Component {
+  render() {
+    const color = site.variables.get(pageBackground);
```

Comment:
What color were you intending on using for the table? This throws an error as `site` and `pageBackground` are undeclared variables.